frontend/src/components/Election/Voting/Thanks.tsx

```diff
@@ -46,7 +46,7 @@ const Thanks = ({ election }) => {
                         }
                         {election.settings.voter_access !== 'closed' &&
                             <Box sx={{ width: '100%', p: 1, px:{xs: 5, sm: 1}  }}>
-                                <ShareButton url={`${window.location.origin}/Election/${election.election_id}`} text={'Invite'} />
```

Comment:
I'm not sure about this one, I think we made a point to change this to say 'Invite' in some places to be clear about what was being shared.

Reply:
Ah ok

When I read Invite I thought it was confusing that it was inconsistent from all the other Share buttons. Invite also has a different meaning to me, and made me think that it wanted me to invite someone specific rather than share it to a general group

I guess "Share Election" would be the most clear, but that's a little long

Reply:
We can stick with share for now, might revisit it later.